### ISSUE TITLE
prokka blast sometimes doesen't find genomes/annotations/prokka/sprot.blast

### ISSUE THREAD

**Issue by SilasK** [tseemann/prokka#361]
Hello, 

I'm running prokka v.13.3 on a cluster. I get repeatedly this error. but when I re-execute prokka, sometimes it workes. I assume it might be because of a filesystem latency error. any ides? 

```
------------- EXCEPTION: Bio::Root::Exception -------------
MSG: Could not read file 'genomes/annotations/prokka/sprot.blast': No such file or directory
STACK: Error::throw
STACK: Bio::Root::Root::throw /home/kiesers/scratch/conda_envs/4e5c88b1/lib/site_perl/5.26.2/Bio/Root/Root.pm:447
STACK: Bio::Root::IO::_initialize_io /home/kiesers/scratch/conda_envs/4e5c88b1/lib/site_perl/5.26.2/Bio/Root/IO.pm:268
STACK: Bio::Root::IO::new /home/kiesers/scratch/conda_envs/4e5c88b1/lib/site_perl/5.26.2/Bio/Root/IO.pm:163
STACK: Bio::SearchIO::new /home/kiesers/scratch/conda_envs/4e5c88b1/lib/site_perl/5.26.2/Bio/SearchIO.pm:185
STACK: Bio::SearchIO::new /home/kiesers/scratch/conda_envs/4e5c88b1/lib/site_perl/5.26.2/Bio/SearchIO.pm:217
STACK: /home/kiesers/scratch/conda_envs/4e5c88b1/bin/prokka:1066
-----------------------------------------------------------
```

**Comment by TANEO-bio** [comment]
Hello.
Maybe I think you wrote script such as below.
```
prokka data1.fasta --outdir prokka --and so on
prokka data2.fasta --outdir prokka --and so on
```
If you put all results in prokka-directory, intermediate file like "sprot.blast" removed by another process. So, you should separate outdir in each processes.

**Comment by tseemann** [comment]
This issue was fixed in c7f9c2fdf3b03fa4aa44c622db41dfffc0bd450c
